deployment.md

```diff
@@ -0,0 +1,67 @@
+
+1. Log in to your GitHub account.
+2. Click the "+" icon in the top right corner of the screen and select "New repository."
+3. Give your repository a name and description.
+4. Choose whether you want your repository to be public or private.
+5. Click the "Create repository" button.
+
+## Step 2: Clone the Repository
+Now that you've created your repository, you need to clone it to your local machine. To do this, follow these steps:
+
+1. Open a terminal on your local machine.
+2. Navigate to the directory where you want to store your project.
+3. Run the following command: `git clone https://github.com/username/repository-name.git` (replace "username" and "repository-name" with your GitHub username and the name of your repository).
+
+## Step 3: Install Dependencies
+Before you can run your project, you need to install its dependencies. To do this, follow these steps:
+
+1. Navigate to the root directory of your project in your terminal.
+2. Run the following command: `npm install` (assuming that you're using npm as your package manager).
+
+## Step 4: Build Your Project
+If your project needs to be built before it can be deployed, you need to do this now. To build your project, follow these steps:
+1. Navigate to the root directory of your project in your terminal.
+2. Run the following command: `npm run build` (assuming that you have a "build" script in your package.json file).
+
+## Step 5: Deploy Your Project
+Now that your project is ready to be deployed, you need to push it to GitHub. To do this, follow these steps:
+
+1. Navigate to the root directory of your project in your terminal.
+2. Run the following commands:
+git add .
+git commit -m "Initial commit"
+git push origin main
+
+(assuming that you're using the main branch as your default branch).
+
+# Setting Up the Development Server
+
+## Step 1: Install Node.js and npm
+Before you can set up the development server, you need to install Node.js and npm on your local machine. To do this, follow these steps:
+
+1. Go to the Node.js website (https://nodejs.org/en/).
+2. Download and install the latest LTS version of Node.js.
+3. Verify that Node.js and npm are installed correctly by running the following commands in your terminal:
+
+node -v
+npm -v
+
+
+## Step 2: Clone the Repository
+Just like when you're deploying your project, you need to clone the repository to your local machine. To do this, follow the same steps as in Step 2 of the previous section.
+
+## Step 3: Install Dependencies
+Before you can run the development server, you need to install its dependencies. To do this, follow the same steps as in Step 3 of the previous section.
+
+## Step 4: Run the Development Server
+Now that you've installed the dependencies, you can run the development server. To do this, follow these steps:
+
+1. Navigate to the root directory of your project in your terminal.
+2. Run the following command: `npm run dev`
```

Comment:
We do not have a `dev` script in our package.json file.
You can validate this from your terminal. Run `npm run dev` and you will get an error.
This should be changes to `npm start` before merging.